CHANGELOG.md

```diff
@@ -308,6 +308,9 @@ changes. Two select improvements:
 
 ### New features
 
+* `jj bookmark create`, `jj bookmark set` and `jj bookmark move` onto a hidden
+  commit make it visible, making the behavior consistent across commands.
```

Comment:
nit: I'm not sure users will understand what "making the behavior consistent across commands" means. I would just remove it. I don't think many users are going to wonder why, so I don't think we need to explain the rationale like we've done a few times before.

Reply:
done